### PR TITLE
Unfreeze setuptools version again, workaround

### PR DESCRIPTION
**Pull request by kxrob**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -46,13 +43,6 @@ jobs:
       run: |
         python setup.py bdist_wheel --skip-build
 
-    # 3.10 dropped support for bdist_wininst. If we had both 3.9 and 3.10 available in a single
```
Comment (mhammond) [inline .github/workflows/main.yml:49]: Why this change? It means we don't get .exe installers from CI.

**Review by mhammond — COMMENTED** [review]
Thanks for following up here, but I don't understand one of the changes.

File: .github/workflows/main.yml (hunk 1 above)
Comment (kxrob) [reply .github/workflows/main.yml:49]: The bdist_wininst has been completely removed recently from setuptools /distutils which is effective with current version of setuptools.  (Commit message of 4. patch: _Remove bdist_wininst from CI (gone in setuptools / distutils) - 
Removed now in pypa/distutils@e28b6193_ )

So its dead until some independent project would continue bdist_wininst or  bdist_msi .
On the build machine you may use an older version of setuptools for some time if needed, or somehow fall back to deprecated legacy distutils in python (where the stub .exe were taken from anyway via setup.py/maybe_fixup_exes() ) ... ?
Or end .exe installers completely?

Comment (mhammond) [reply .github/workflows/main.yml:49]: Sorry the the delay and thanks for the explanation! Download stats of the .exe from github are quite minor compared to pypi, so I think I'll open an issue to phase them out, but might as well do that as late as practical. My local environment can still .exe installers, so 👍 and thanks!

**Pull request merged**